examples/chef/devices/rootnode_waterleakdetector_0b067acfa3.zap

```diff
@@ -2071,12 +2071,18 @@
       "id": 2,
       "name": "Anonymous Endpoint Type",
       "deviceTypeRef": {
-        "code": 67,
+        "code": 17,
         "profileId": 259,
-        "label": "MA-water-leak-detector",
-        "name": "MA-water-leak-detector"
+        "label": "MA-powersource",
+        "name": "MA-powersource"
```

Comment:
Surprised the device type of the endpoint changes label name by addition of just 1 power source device type. Is there a way to order power source after leak detector?

Reply:
I think there is an issue with ZAP because even I set `Matter Water Leak Detector (0x0043)` as a primary device and save that file:

![image](https://github.com/user-attachments/assets/326092a8-a9e9-4d4a-874b-c7882e78a37d)

But in the file it is not correct. When I reopen the file. The order has changed.
![image](https://github.com/user-attachments/assets/77f9f0ea-0b69-4483-92bb-29eefb9c6472)

Reply:
I've opened an issue [[BUG] ZAP tool --> Primary Device is not correctly saved to zap file #35437](https://github.com/project-chip/connectedhomeip/issues/35437)

Reply:
Fixed by commit [#6df7dc1](https://github.com/project-chip/connectedhomeip/pull/35410/commits/6df7dc121d5631a3ea90318210591acd31078ef0)